src/Execution/Utils/ModelKey.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Nuwave\Lighthouse\Execution\Utils;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+/**
+ * Create a model key that concatenates the models fully-qualified class
+ * name and key or composite key.
+ *
+ * For example: App\Model\Post:1
+ */
+class ModelKey
+{
+    public static function build(Model $model): string
+    {
+        return Collection::make([get_class($model)])
```

Comment:
```suggestion
        return (new Collection([get_class($model)]))
```

Reply:
Actually, can you write that function without collections? This code instantiates three Collection objects to do something quite simple. Performance matters.

Reply:
You got it.